odxtools/envdata.py

```diff
@@ -17,6 +17,7 @@
 @dataclass
 class EnvironmentData(BasicStructure):
     """This class represents Environment Data that describes the circumstances in which the error occurred."""
+    dtc_values: List[int]
```

Comment:
Where did this come from?

Reply:
this is new. I noticed that the XSD features a "DTC-VALUES" tag, but the python class didn't. I'll factor it out...

Reply:
actually, the above is inaccurate: currently, `EnvironmentData` is a dataclass that defines `__init__()` and `dtc_values` was only created by the latter. I suppose the manual constructor ought to be removed...

Reply:
... but that is currently not so easy because the base class (`BasicStructure`) is currently not yet a dataclass. let's just keep that kludge for now...